src/matrix/matrix.ts

```diff
@@ -71,7 +71,7 @@ export function matrixForQuery (
     header.setAttribute('style', 'padding: 0.3em;')
     header.textContent = utils.label(y1) // first approximation
     if (y1.termType === 'NamedNode') {
-      kb.fetcher.nowOrWhenFetched(y1.uri.split('#')[0], undefined, function (
+      kb.fetcher!.nowOrWhenFetched(y1.uri.split('#')[0], undefined, function (
```

Comment:
Can we rely on that the fetcher is available. If so, can we reflect that in the type?

Reply:
Yes.  There is a subclass of Store, LiveStore, (IIRC) which has fetcher and updater.

Reply:
So, should the type in solid-logic changed to LiveStore @michielbdejong